### DISCUSSION TITLE
Kernel not working

### DISCUSSION THREAD

**Discussion by mahnoor-byte** [AammarTufail/python_ka_chilla2023#68] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/121869407/212708451-b0d24f07-d132-4607-8051-770516fe773e.png)

It is taking too much time to connect to kernel. How can I resolve it? 
It was working fine. I reinstalled python and now this is not working.

**Comment by MuhammadHammad4127** [comment, marked as answer]
Current Version Download krein jo 3.11 hy

**Reply by mahnoor-byte** [reply]
I tried that as well previously but didn't work.

I have uninstalled Jupyter and then reinstalled. Now it is working. Thank you.

**Comment by AftabHaider680** [comment]
![1](https://user-images.githubusercontent.com/123288921/215269524-2645a684-7101-4dc5-b1a5-88a13c6e5c90.JPG)
Plaese help me to solve this issue, i try many times even uninstall and then install both VS code and Python as well as jupyter extensions but every time problem is same @MuhammadHammad4127

**Reply by Sahibzada18** [reply]
Uninstall your python and remove it from variable path hope it work out for you
